src/select.scss

```diff
@@ -81,34 +96,56 @@ $block: #{$fd-namespace}-select;
     }
 
     &.is-readonly {
-      padding-right: $fd-select-padding-x;
-      padding-left: $fd-select-padding-x;
+      .#{$block}__text-content {
+        padding-right: $fd-select-padding-x;
+        padding-left: $fd-select-padding-x;
+      }
+
       cursor: text;
     }
   }
 
   &__text {
+    @include fd-reset();
```

Comment:
I don't see where we use this class

Reply:
it was there previously, so instead fo causing breaking change, I changed everything to __Text and not __text-content

Reply:
But renaming `__text` to `__text-content` is a Breaking change